rules/eslint/style/walmart.js

```diff
@@ -22,6 +22,8 @@ module.exports = {
     "func-names": 0,
     // enforces use of function declarations or expressions
     "func-style": [2, "expression"],
+    // this option enforces minimum and maximum identifier lengths (variable names, property names etc.)
+    "id-length": 0,
```

Comment:
I actually wish it was the other way around. I prefer long names to short ones like `x`, `fi` and others. I've even seen people use acronymns :(.